docs/notebooks/zero_inflated_regression.ipynb

```diff
@@ -0,0 +1,1905 @@
+{
```

Comment:
<p>Poisson better than "poisson" because it's a surname</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/725/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>